src/Database/DataFeed.php

```diff
@@ -228,9 +245,9 @@ protected function processCollection()
             $cleanQuery = $cleanQuery->select(Db::raw($idSelect))
                 ->addSelect(Db::raw($tagSelect))
                 ->addSelect(Db::raw($sortSelect));
-            
+
             // Union this query with the previous one
-            if ($lastQuery) {
+            if ($lastQuery !== null) {
```

Comment:
What was the reason for this change?

Reply:
More strict check, nothing else

Reply:
If you want a more strict check then check to see if it's actually the object that it's expected to be, one that supports the `union()` and `unionAll()` methods.  Checking `if ($lastQuery)` vs `if ($lastQuery !== null)` provides no benefit and actually is worse because it allows falsey values like an empty array, string, or false boolean to slip through the cracks.

Reply:
I could remove it :)

Reply:
Revert it or make it properly strict, whichever you prefer. !== null won't be merged though 😉

Reply:
@LukeTowers reverted null check